available_images.md

```diff
@@ -0,0 +1,140 @@
+|  			 EU (Stockholm) 			            |  			 eu-north-1 			     |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-north-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			       |
+|  			 EU (Ireland) 			              |  			 eu-west-1 			      |  			 Available 			         |  			 Available 			                   |  			 763104351884.dkr.ecr.eu-west-1.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 EU (London) 			               |  			 eu-west-2 			      |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-west-2.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 EU (Paris) 			                |  			 eu-west-3 			      |  			 Available 			         |  			 None 			                        |  			 763104351884.dkr.ecr.eu-west-3.amazonaws.com/&lt;repository-name>:&lt;image-tag> 			        |
+|  			 China (Beijing) 			           |  			 cn-north-1 			     |  			 Available 			         |  			 None 			                        |  			 727897471807.dkr.ecr.cn-north-1.amazonaws.com.cn/&lt;repository-name>:&lt;image-tag> 			    |
+|  			 China (Ningxia) 			           |  			 cn-northwest-1 			 |  			 Available 			         |  			 None 			                        |  			 727897471807.dkr.ecr.cn-northwest-1.amazonaws.com.cn/&lt;repository-name>:&lt;image-tag> |
+
+
+Note: **eu-north-1** is available starting in the version 2.0 release.
+As a result, MXNet-1.4.0 is not available in this region.
+
+ECR is a regional service and the Image table contains the URLs for
+**us-east-1** images. To pull from one of the regions mentioned
+previously, insert the region in the repository URL following this
+example:
+
+
+
+     763104351884.dkr.ecr.<region>.amazonaws.com/tensorflow-training:1.15.2-cpu-py27-ubuntu18.04
+
+**Important**
+
+You must login to access to the DLC image repository before pulling
+the image. Ensure your CLI is up to date using the steps in [Installing the current AWS CLI Version](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html#install-tool-bundled)
+    Then, specify your region and its corresponding ECR Registry from
+    the previous table in the following command:
+
+
+
+        aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
+
+You can then pull these Docker images from ECR by running:
+
+
+
+    docker pull <name of container image>
+
+ General Framework Containers
+============================
+
+To use the following table, select your desired framework, as well as
+the kind of job you're starting, and the desired Python version. Your
+job type is either ``training`` or ``inference``. Your Python version is
+either ``py27`` or ``py36``. Plug this information into the replaceable
```

Comment:
We have py37 versions for TF 1.15.2 and TF 2.2, as is also mentioned in the table so this statement needs amends.

Reply:
Done